storage/tianmu/common/exception.h

```diff
@@ -64,22 +64,22 @@ inline bool IsWarning(ErrorCode tianmu_rc) {
 
 class TianmuError {
  public:
-  TianmuError(ErrorCode tianmu_error_code = ErrorCode::SUCCESS) : ec(tianmu_error_code) {}
-  TianmuError(ErrorCode tianmu_error_code, std::string message) : ec(tianmu_error_code), message(message) {}
-  TianmuError(const TianmuError &tianmu_e) : ec(tianmu_e.ec), message(tianmu_e.message) {}
+  TianmuError(ErrorCode tianmu_error_code = ErrorCode::SUCCESS) : ec_(tianmu_error_code) {}
+  TianmuError(ErrorCode tianmu_error_code, std::string message) : ec_(tianmu_error_code), message_(message) {}
+  TianmuError(const TianmuError &tianmu_e) : ec_(tianmu_e.ec_), message_(tianmu_e.message_) {}
   TianmuError &operator=(const TianmuError &) = default;
-  operator ErrorCode() { return ec; }
-  ErrorCode GetErrorCode() { return ec; }
-  bool operator==(ErrorCode tianmu_ec) { return tianmu_ec == ec; }
+  operator ErrorCode() { return ec_; }
+  ErrorCode GetErrorCode() { return ec_; }
+  bool operator==(ErrorCode tianmu_ec) { return tianmu_ec == ec_; }
   const std::string &Message() {
-    if (!message.empty())
-      return message;
-    return error_messages[static_cast<int>(ec)];
+    if (!message_.empty())
+      return message_;
+    return error_messages[static_cast<int>(ec_)];
   }
 
  private:
-  ErrorCode ec;
-  std::string message;
+  ErrorCode ec_;
```

Comment:
using `err_code_`, not just with underscore。 this pr main goal is to improve the code readibity. not just add some underscores.